rollup.config.js

```diff
@@ -26,11 +26,13 @@ export default [
       commonjs({
         include: /node_modules/,
         sourceMap: true,
+        transformMixedEsModules: true,
```

Comment:
is this because we can finally start using ESM versions of Otel?

Reply:
rollup automatically started pulling esm versions, but as lodash-merge still requires using commonjs style require, this flag is needed to still check esm files for requires